warehouse/query-core/src/test/java/datawave/query/config/ShardQueryConfigurationTest.java

```diff
@@ -516,47 +517,6 @@ public void testIsTldQuery() {
         Assert.assertTrue(config.isTldQuery());
     }
 
-    /**
-     * This test will fail if a new variable is added improperly to the ShardQueryConfiguration
-     * <p>
-     * If a configuration option was added do the following as appropriate
-     * <ol>
-     * <li>Add getter/setters to the ShardQueryLogic that delegate to the ShardQueryConfiguration</li>
```

Comment:
This guard should be disabled instead of removed. While it doesn't do much in the way of actual testing it does alert a developer that adding a new config option requires certain steps. 

I'm thinking through what a better solution would look like.

Reply:
We definitly need a better solution.  We could actually verify that there are setter and getter methods for each of the properties, and could actually verify they are copied in the cloning constructor by using reflection.